backend/hct_mis_api/apps/account/permissions.py

```diff
@@ -345,7 +346,14 @@ def has_permission(cls, info: Any, **kwargs: Any) -> bool:
     return XDPerm
 
 
-class BaseNodePermissionMixin:
+class AdminUrlNodeMixin:
+    admin_url = graphene.String()
+
+    def resolve_admin_url(self, info: Any, **kwargs: Any) -> graphene.String:
+        return self.admin_url
+
+
+class BaseNodePermissionMixin(AdminUrlNodeMixin):
```

Comment:
I'm not sure that this class should inherit from AdminUrlNodeMixin, this is a different topic